### PR TITLE
[WiP] Contactimport #297

### PR DESCRIPTION
**Pull request by wischweh**

### CODE REVIEW

**Comment by audax** [comment]
Which of these commits fixes the contact import?

**Comment by audax** [comment]
Please rebase the PR on master, so that only 2 commits remain: 1 where you implement the new test and 1 where you fix the test.

There is way to much clutter in those commits that prevents me from reviewing it properly.

File: qabelbox/src/main/res/values/servers.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="blockServer" translatable="false">https://block.qabel.org</string>
-    <string name="accountingServer" translatable="false">https://accounting.qabel.org</string>
-    <string name="dropServer" translatable="false">https://drop.qabel.org</string>
+	<string name="blockServer" translatable="false">https://block.qabel.org</string>
+	<string name="accountingServer" translatable="false">https://accounting.qabel.org</string>
```
Comment (julianseeger) [inline qabelbox/src/main/res/values/servers.xml]: tabs?

**Comment by wischweh** [comment]
continued work in https://github.com/Qabel/qabel-android/pull/454 with cleaner history

**Pull request closed without merging**